quiz.js

```diff
@@ -0,0 +1,64 @@
+"use strict"
+
+	CarLot = (function(oldCarLot){
+
+    oldCarLot.displayCars = function(inventory){
+        var inCars = document.getElementsByClassName("row")[0];
```

Comment:
Also, these rows should be created in your js file not hardcoded in the DOM. Remember to use bootstrap to style the rows, and make sure only three cards can exist per row.